src/transform.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+module.exports = {
+    tile: transformTile,
+    point: transformPoint
+};
+
+/**
+ * Transforms the coordinates each feature in the given tile from projected
+ * space into tile space.
+ * @param {object} tile
+ * @param {number} extent The extent of the tile (in tile space units)
+ */
```

Comment:
Let's use simple minimal comments instead of JSDoc for consistency for now; if I eventually decide adding JSDoc, I'd do it separately but on the whole project.